### DISCUSSION TITLE
What models are you using, or hoping to use, with TensorRT?

### DISCUSSION THREAD

**Discussion by jaybdub** [NVIDIA-AI-IOT/torch2trt#531] [Models]
Hi All,

We're interested in hearing how you're using, or hoping to use torch2trt.  This helps us prioritize features, or suggest workarounds if they exist.  Please feel free to share here with

Using: \<model name\>, \<model URL\>
Notes: ...

or

Hoping to Use: \<model name\>, \<model URL\>
Notes: ...

If you find a model in this thread that matches you're use case, simply +1 that comment.

Look forward to hearing from you!

Best,
John

**Comment by Boatsure** [comment]
Hoping to Use: CenterNet, [https://openaccess.thecvf.com/content_ICCV_2019/html/Duan_CenterNet_Keypoint_Triplets_for_Object_Detection_ICCV_2019_paper.html](url)

**Reply by jaybdub** [reply]
Thanks @surefyyq !  Hope to take a look soon.  Please feel to reach out if you have other suggestions as well.

**Reply by zzz3bbb3** [reply]
You can refer to this repo: [https://github.com/nanmi/CenterNet-trt](https://github.com/nanmi/CenterNet-trt)

**Comment by YoushaaMurhij** [comment]
I hope to use [CenterPoint](https://github.com/tianweiy/CenterPoint) (CVPR [paper](https://arxiv.org/abs/2006.11275))
The model contains multiple modules and a multiple inputs/outputs(heads) .. including dictionary data structures.

**Reply by sunziq322** [reply]
I want to deploy a customed  yolov5m.pt on jetson NX.
Jetpack 4.5, cuda 10.2, cudnn8.0, tensorrt 7.3.xx
Is that possible to convert it to .engine?

**Comment by Fan9** [comment]
Bert  !!!!!

**Reply by Deep-learning999** [reply]
Hoping to Use: YOLOP SLOWFAST SLOWONLY ST-GCN YOLOV5 DEEPSORT

**Comment by imoneoi** [comment]
Hope to use: DeepLabv3+

**Reply by FreedomLiX** [reply]
Write codes in C++ API

**Comment by maronuu** [comment]
Hoping to Use: [YOLOR](https://github.com/WongKinYiu/yolor), [paper](https://arxiv.org/abs/2105.04206)

**Reply by rodja** [reply]
@maronuu has made some progress with YoloR and described it in https://github.com/NVIDIA-AI-IOT/torch2trt/issues/565. Now it seems the `torch.nn.parameter.Parameter` converter is missing...

**Comment by tianyu06030020** [comment]
hoping for: torch.nn.ReflectionPad2d(), which is a padding implemention from pytorch. thx

**Comment by roulbac** [comment]
Hoping to use Spatial Transformer Networks : [link](https://pytorch.org/tutorials/intermediate/spatial_transformer_tutorial.html)

**Comment by prabhd33ps** [comment]
Hoping to use DSFD [Official Link](https://github.com/Tencent/FaceDetection-DSFD)

**Comment by xunleiwu** [comment]
Hoping to Use: DeTR, https://github.com/facebookresearch/detr
Thank you very much.
We are looking for the conversion plugin of Multi-Head Attention used in DeTR.

**Comment by peria1** [comment]
Hoping to Use: Detectron2,  https://github.com/facebookresearch/detectron2

We have had trouble with other routes to production, so we are taking a look at TensorRT.

**Comment by qzsrh** [comment]
Hoping to Use: yolact
https://github.com/KevinJia1212/yolact_cityscapes_550

**Comment by iNeil77** [comment]
Hoping to use DeiT variant of VIT
https://arxiv.org/abs/2012.12877

**Comment by Deep-learning999** [comment]
Hoping to Use: YOLOP SLOWFAST SLOWONLY ST-GCN YOLOV5 DEEPSORT

**Reply by Deep-learning999** [reply]
https://github.com/yysijie/st-gcn

**Comment by oliver0922** [comment]
mobilenetv2 Please :)
Link: https://pytorch.org/vision/master/_modules/torchvision/models/mobilenetv2.html

**Reply by SJavad** [reply]
I also have to convert ssd mobile net to tensorrt model for inferencing on jetson tx1

**Comment by Hong-Bo** [comment]
StyleGAN2-pytorch Danke!
Link: https://github.com/rosinality/stylegan2-pytorch

**Comment by flclain** [comment]
LaneGCN and DenseTNT Please ^_^
https://github.com/uber-research/LaneGCN
https://github.com/Tsinghua-MARS-Lab/DenseTNT

**Comment by FreedomLiX** [comment]
Bert, models with attention,&nbsp;



---Original---
From: ***@***.***&gt;
Date: Thu, Dec 30, 2021 17:36 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [NVIDIA-AI-IOT/torch2trt] What models are you using, or hoping touse, with TensorRT? (Discussion #531)




 
LaneGCN and DenseTNT Please ^_^
 https://github.com/uber-research/LaneGCN
 https://github.com/Tsinghua-MARS-Lab/DenseTNT
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by jianghaijun007** [comment]
MonoFlex, Please ^_^
https://github.com/zhangyp15/MonoFlex

**Comment by aligoglos** [comment]
Hoping to Use: yolox
[https://github.com/Megvii-BaseDetection/YOLOX](https://github.com/Megvii-BaseDetection/YOLOX)

**Comment by FreedomLiX** [comment]
Hi,&nbsp; NVIDIA-AI-IOT :

&nbsp;&nbsp;&nbsp;&nbsp; what I hoping to is TensorRT -API&nbsp; develop ( AdaptAvgPooling&nbsp; , Deformable Convolution ) function.





------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "NVIDIA-AI-IOT/torch2trt"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年4月21日(星期四) 晚上9:25
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [NVIDIA-AI-IOT/torch2trt] What models are you using, or hoping to use, with TensorRT? (Discussion #531)





 
Hoping to Use: yolox
 https://github.com/Megvii-BaseDetection/YOLOX

**Comment by jcyhcs** [comment]
hope to use: SiamRPN++
https://github.com/STVIR/pysot
please!

**Comment by FreedomLiX** [comment]
its' easy！

---Original---
From: ***@***.***&gt;
Date: Tue, May 3, 2022 09:31 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [NVIDIA-AI-IOT/torch2trt] What models are you using, or hoping touse, with TensorRT? (Discussion #531)




 
I want to deploy a customed  yolov5m.pt on jetson NX.
 Jetpack 4.5, cuda 10.2, cudnn8.0, tensorrt 7.3.xx
 Is that possible to convert it to .engine?

**Comment by sunziq322** [comment]
THX





发件人：FreedomLi ***@***.***>
发送日期：2022-05-03 09:45:31
收件人：NVIDIA-AI-IOT/torch2trt ***@***.***>
抄送人：sunziq322 ***@***.***>,Comment ***@***.***>
主题：Re: [NVIDIA-AI-IOT/torch2trt] What models are you using, or hoping to use, with TensorRT? (Discussion #531)

its' easy！

 ---Original---
 From: ***@***.***&gt;
 Date: Tue, May 3, 2022 09:31 AM
 To: ***@***.***&gt;;
 Cc: ***@***.******@***.***&gt;;
 Subject: Re: [NVIDIA-AI-IOT/torch2trt] What models are you using, or hoping touse, with TensorRT? (Discussion #531)





 I want to deploy a customed yolov5m.pt on jetson NX.
 Jetpack 4.5, cuda 10.2, cudnn8.0, tensorrt 7.3.xx
 Is that possible to convert it to .engine?

**Comment by sunziq322** [comment]
I am using a customed yolov5m.pt, origional model has 319 layers, I added a CABlock, Conv module applies GhostConv, so the
current module has 422 layers,


# YOLOv5 by Ultralytics, GPL-3.0 license

# Parameters
nc: 1 # number of classes
depth_multiple: 0.67 # model depth multiple
width_multiple: 0.75 # layer channel multiple
anchors:
- [10,13, 16,30, 33,23] # P3/8
- [30,61, 62,45, 59,119] # P4/16
- [116,90, 156,198, 373,326] # P5/32

# YOLOv5 v6.0 backbone
backbone:
# [from, number, module, args]
[[-1, 1, Conv, [64, 6, 2, 2]], # 0-P1/2
[-1, 1, GhostConv, [128, 3, 2]], # 1-P2/4
[-1, 3, C3, [128]],
[-1, 1, GhostConv, [256, 3, 2]], # 3-P3/8
[-1, 6, C3, [256]],
[-1, 1, GhostConv, [512, 3, 2]], # 5-P4/16
[-1, 9, C3, [512]],
[-1, 1, GhostConv, [1024, 3, 2]], # 7-P5/32
[-1, 3, C3, [1024]],
[-1, 1, CABlock, [1024, 32]],
[-1, 1, SPPF, [1024, 5]], # 9
]

# YOLOv5 v6.0 head
head:
[[-1, 1, GhostConv, [512, 1, 1]],
[-1, 1, nn.Upsample, [None, 2, 'nearest']],
[[-1, 6], 1, Concat, [1]], # cat backbone P4
[-1, 3, C3, [512, False]], # 13

[-1, 1, GhostConv, [256, 1, 1]],
[-1, 1, nn.Upsample, [None, 2, 'nearest']],
[[-1, 4], 1, Concat, [1]], # cat backbone P3
[-1, 3, C3, [256, False]], # 17 (P3/8-small)

[-1, 1, GhostConv, [256, 3, 2]],
[[-1, 15, 6], 1, Concat, [1]], # cat head P4
[-1, 3, C3, [512, False]], # 20 (P4/16-medium)

[-1, 1, GhostConv, [512, 3, 2]],
[[-1, 11], 1, Concat, [1]], # cat head P5
[-1, 3, C3, [1024, False]], # 23 (P5/32-large)

[[18, 21, 24], 1, Detect, [nc, anchors]], # Detect(P3, P4, P5)
]




Please info me some easy ways to prune it. Thank you very much.



发件人：FreedomLi ***@***.***>
发送日期：2022-05-03 09:45:31
收件人：NVIDIA-AI-IOT/torch2trt ***@***.***>
抄送人：sunziq322 ***@***.***>,Comment ***@***.***>
主题：Re: [NVIDIA-AI-IOT/torch2trt] What models are you using, or hoping to use, with TensorRT? (Discussion #531)

its' easy！

 ---Original---
 From: ***@***.***&gt;
 Date: Tue, May 3, 2022 09:31 AM
 To: ***@***.***&gt;;
 Cc: ***@***.******@***.***&gt;;
 Subject: Re: [NVIDIA-AI-IOT/torch2trt] What models are you using, or hoping touse, with TensorRT? (Discussion #531)





 I want to deploy a customed yolov5m.pt on jetson NX.
 Jetpack 4.5, cuda 10.2, cudnn8.0, tensorrt 7.3.xx
 Is that possible to convert it to .engine?

**Comment by Yshelgi** [comment]
efficientdet!!!

**Comment by FreedomLiX** [comment]
easy！



---Original---
From: ***@***.***&gt;
Date: Wed, May 11, 2022 16:21 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [NVIDIA-AI-IOT/torch2trt] What models are you using, or hoping touse, with TensorRT? (Discussion #531)




 
efficientdet!!!

**Comment by FreedomLiX** [comment]
please , info your CABlock details.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "NVIDIA-AI-IOT/torch2trt"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年5月5日(星期四) 凌晨4:05
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [NVIDIA-AI-IOT/torch2trt] What models are you using, or hoping to use, with TensorRT? (Discussion #531)





 I am using a customed yolov5m.pt, origional model has 319 layers, I added a CABlock, Conv module applies GhostConv, so the
 current module has 422 layers,
 
 
 # YOLOv5 by Ultralytics, GPL-3.0 license
 
 # Parameters
 nc: 1 # number of classes
 depth_multiple: 0.67 # model depth multiple
 width_multiple: 0.75 # layer channel multiple
 anchors:
 - [10,13, 16,30, 33,23] # P3/8
 - [30,61, 62,45, 59,119] # P4/16
 - [116,90, 156,198, 373,326] # P5/32
 
 # YOLOv5 v6.0 backbone
 backbone:
 # [from, number, module, args]
 [[-1, 1, Conv, [64, 6, 2, 2]], # 0-P1/2
 [-1, 1, GhostConv, [128, 3, 2]], # 1-P2/4
 [-1, 3, C3, [128]],
 [-1, 1, GhostConv, [256, 3, 2]], # 3-P3/8
 [-1, 6, C3, [256]],
 [-1, 1, GhostConv, [512, 3, 2]], # 5-P4/16
 [-1, 9, C3, [512]],
 [-1, 1, GhostConv, [1024, 3, 2]], # 7-P5/32
 [-1, 3, C3, [1024]],
 [-1, 1, CABlock, [1024, 32]],
 [-1, 1, SPPF, [1024, 5]], # 9
 ]
 
 # YOLOv5 v6.0 head
 head:
 [[-1, 1, GhostConv, [512, 1, 1]],
 [-1, 1, nn.Upsample, [None, 2, 'nearest']],
 [[-1, 6], 1, Concat, [1]], # cat backbone P4
 [-1, 3, C3, [512, False]], # 13
 
 [-1, 1, GhostConv, [256, 1, 1]],
 [-1, 1, nn.Upsample, [None, 2, 'nearest']],
 [[-1, 4], 1, Concat, [1]], # cat backbone P3
 [-1, 3, C3, [256, False]], # 17 (P3/8-small)
 
 [-1, 1, GhostConv, [256, 3, 2]],
 [[-1, 15, 6], 1, Concat, [1]], # cat head P4
 [-1, 3, C3, [512, False]], # 20 (P4/16-medium)
 
 [-1, 1, GhostConv, [512, 3, 2]],
 [[-1, 11], 1, Concat, [1]], # cat head P5
 [-1, 3, C3, [1024, False]], # 23 (P5/32-large)
 
 [[18, 21, 24], 1, Detect, [nc, anchors]], # Detect(P3, P4, P5)
 ]
 
 
 
 
 Please info me some easy ways to prune it. Thank you very much.
 
 
 
 发件人：FreedomLi ***@***.***&gt;
 发送日期：2022-05-03 09:45:31
 收件人：NVIDIA-AI-IOT/torch2trt ***@***.***&gt;
 抄送人：sunziq322 ***@***.***&gt;,Comment ***@***.***&gt;
 主题：Re: [NVIDIA-AI-IOT/torch2trt] What models are you using, or hoping to use, with TensorRT? (Discussion #531)
 
 its' easy！
 
  ---Original---
  From: ***@***.***&amp;gt;
  Date: Tue, May 3, 2022 09:31 AM
  To: ***@***.***&amp;gt;;
  Cc: ***@***.******@***.***&amp;gt;;
  Subject: Re: [NVIDIA-AI-IOT/torch2trt] What models are you using, or hoping touse, with TensorRT? (Discussion #531)
 
 
 
 
 
  I want to deploy a customed yolov5m.pt on jetson NX.
  Jetpack 4.5, cuda 10.2, cudnn8.0, tensorrt 7.3.xx
  Is that possible to convert it to .engine?

**Comment by Jae-1409** [comment]
U2Net, please
https://github.com/xuebinqin/U-2-Net

**Comment by sunziq322** [comment]
Sorry for late reply.
I am using tensorrt 8.0 on Jetson nx, and I removed that CABlock.

I converted pt model to onnx, and then to trt. Now I integrated trt into deepstream, it worked.
As for Api, I used Yolo_on_Jetson after trival modification.


Thank you very much.






发件人：FreedomLi ***@***.***>
发送日期：2022-05-30 15:37:22
收件人：NVIDIA-AI-IOT/torch2trt ***@***.***>
抄送人：sunziq322 ***@***.***>,Comment ***@***.***>
主题：Re: [NVIDIA-AI-IOT/torch2trt] What models are you using, or hoping to use, with TensorRT? (Discussion #531)

please , info your CABlock details.




 ------------------&nbsp;原始邮件&nbsp;------------------
 发件人: "NVIDIA-AI-IOT/torch2trt" ***@***.***&gt;;
 发送时间:&nbsp;2022年5月5日(星期四) 凌晨4:05
 ***@***.***&gt;;
 ***@***.******@***.***&gt;;
 主题:&nbsp;Re: [NVIDIA-AI-IOT/torch2trt] What models are you using, or hoping to use, with TensorRT? (Discussion #531)





 I am using a customed yolov5m.pt, origional model has 319 layers, I added a CABlock, Conv module applies GhostConv, so the
 current module has 422 layers,


 # YOLOv5 by Ultralytics, GPL-3.0 license

 # Parameters
 nc: 1 # number of classes
 depth_multiple: 0.67 # model depth multiple
 width_multiple: 0.75 # layer channel multiple
 anchors:
 - [10,13, 16,30, 33,23] # P3/8
 - [30,61, 62,45, 59,119] # P4/16
 - [116,90, 156,198, 373,326] # P5/32

 # YOLOv5 v6.0 backbone
 backbone:
 # [from, number, module, args]
 [[-1, 1, Conv, [64, 6, 2, 2]], # 0-P1/2
 [-1, 1, GhostConv, [128, 3, 2]], # 1-P2/4
 [-1, 3, C3, [128]],
 [-1, 1, GhostConv, [256, 3, 2]], # 3-P3/8
 [-1, 6, C3, [256]],
 [-1, 1, GhostConv, [512, 3, 2]], # 5-P4/16
 [-1, 9, C3, [512]],
 [-1, 1, GhostConv, [1024, 3, 2]], # 7-P5/32
 [-1, 3, C3, [1024]],
 [-1, 1, CABlock, [1024, 32]],
 [-1, 1, SPPF, [1024, 5]], # 9
 ]

 # YOLOv5 v6.0 head
 head:
 [[-1, 1, GhostConv, [512, 1, 1]],
 [-1, 1, nn.Upsample, [None, 2, 'nearest']],
 [[-1, 6], 1, Concat, [1]], # cat backbone P4
 [-1, 3, C3, [512, False]], # 13

 [-1, 1, GhostConv, [256, 1, 1]],
 [-1, 1, nn.Upsample, [None, 2, 'nearest']],
 [[-1, 4], 1, Concat, [1]], # cat backbone P3
 [-1, 3, C3, [256, False]], # 17 (P3/8-small)

 [-1, 1, GhostConv, [256, 3, 2]],
 [[-1, 15, 6], 1, Concat, [1]], # cat head P4
 [-1, 3, C3, [512, False]], # 20 (P4/16-medium)

 [-1, 1, GhostConv, [512, 3, 2]],
 [[-1, 11], 1, Concat, [1]], # cat head P5
 [-1, 3, C3, [1024, False]], # 23 (P5/32-large)

 [[18, 21, 24], 1, Detect, [nc, anchors]], # Detect(P3, P4, P5)
 ]




 Please info me some easy ways to prune it. Thank you very much.



 发件人：FreedomLi ***@***.***&gt;
 发送日期：2022-05-03 09:45:31
 收件人：NVIDIA-AI-IOT/torch2trt ***@***.***&gt;
 抄送人：sunziq322 ***@***.***&gt;,Comment ***@***.***&gt;
 主题：Re: [NVIDIA-AI-IOT/torch2trt] What models are you using, or hoping to use, with TensorRT? (Discussion #531)

 its' easy！

 ---Original---
 From: ***@***.***&amp;gt;
 Date: Tue, May 3, 2022 09:31 AM
 To: ***@***.***&amp;gt;;
 Cc: ***@***.******@***.***&amp;gt;;
 Subject: Re: [NVIDIA-AI-IOT/torch2trt] What models are you using, or hoping touse, with TensorRT? (Discussion #531)





 I want to deploy a customed yolov5m.pt on jetson NX.
 Jetpack 4.5, cuda 10.2, cudnn8.0, tensorrt 7.3.xx
 Is that possible to convert it to .engine?

**Comment by kishcs** [comment]
Hoping to use `CRAFT-PyTorch`, [Repo](https://github.com/clovaai/CRAFT-pytorch)
Notes: I converted `craft_mlt_25k.pth` to `craft_trt.pth`/`craft.trt`. But unable to load the converted trt model in `netron`.

**Comment by dingjingzhen** [comment]
Hoping to Use: BGNet,
https://github.com/3DCVdeveloper/BGNet

**Comment by mansirthd** [comment]
Hoping to use ByteTrack https://github.com/ifzhang/ByteTrack

I was wondering if I have 100 Jetson AGX devices with the exact same specification, then do you I need to run torch2trt on each of these devices or can I run it on one device and use the output model.engine on the other devices for best performance.

**Comment by dinushazoomi** [comment]
Hoping to use ViTPose with Jetson AGX Orin: https://github.com/ViTAE-Transformer/ViTPose

I used it's PyTorch to ONNX conversion code and then use trtexec to convert to an engine file, But the output heat map is different when using tensorRT inference,

**Comment by FreedomLiX** [comment]
please take more net detials，or the net paper！



---Original---
From: ***@***.***&gt;
Date: Wed, Oct 12, 2022 13:59 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [NVIDIA-AI-IOT/torch2trt] What models are you using, or hoping touse, with TensorRT? (Discussion #531)




 
Hoping to use ViTPose with Jetson AGX Orin: https://github.com/ViTAE-Transformer/ViTPose
 
I used it's PyTorch to ONNX conversion code and then use trtexec to convert to an engine file, But the output heat map is different when using tensorRT inference,

**Comment by FreedomLiX** [comment]
Take it easy



---Original---
From: ***@***.***&gt;
Date: Wed, Nov 9, 2022 15:52 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [NVIDIA-AI-IOT/torch2trt] What models are you using, or hoping to use, with TensorRT? (Discussion #531)




 
YOLOv7-pose

**Comment by FreedomLiX** [comment]
code link



---Original---
From: ***@***.***&gt;
Date: Wed, Nov 9, 2022 15:52 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [NVIDIA-AI-IOT/torch2trt] What models are you using, or hoping to use, with TensorRT? (Discussion #531)




 
YOLOv7-pose

**Comment by TiesenX** [comment]
Hoping to Use: Detic
https://github.com/facebookresearch/Detic

**Comment by FreedomLiX** [comment]
please info the net details!



---Original---
From: "Le Minh ***@***.***&gt;
Date: Mon, Feb 6, 2023 12:38 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [NVIDIA-AI-IOT/torch2trt] What models are you using, or hoping touse, with TensorRT? (Discussion #531)




 
Hoping to Use: Detic
 https://github.com/facebookresearch/Detic

**Comment by blue-q** [comment]
Hoping to Use: FIDTM
https://github.com/dk-liang/FIDTM

**Comment by foemre** [comment]
Hoping to use: MiDaS
https://github.com/isl-org/MiDaS

**Comment by ziqin-h** [comment]
Hoping to use: Bot-Sort
https://github.com/NirAharon/BoT-SORT

**Comment by RemanentQuill76** [comment]
Hoping to use: Yolov8 detect 
https://hub.ultralytics.com/models/Wif16r2vflrJ1rc30ca2

**Comment by fabricecarles** [comment]
Hoping to Use MiDaS https://pytorch.org/hub/intelisl_midas_v2/
Hoping to Use RAFT https://pytorch.org/vision/main/models/raft.html

**Comment by lofyol** [comment]
yolov8-pose

**Comment by kais-bedioui** [comment]
@lofyol Hi, you might want to keep an eye on this repo:
https://github.com/marcoslucianops/DeepStream-Yolo-Pose
The author mentioned lately (on Linkedin at least) that he will start
working on Yolo-Pose
Good Luck

**Comment by soerenbrockmann** [comment]
How to convert a model that has not supported operations? Is there any example for it? Documentation is quite hard to understand...

**Comment by mukul74** [comment]
Hoping to Use: [Retinexformer](https://github.com/caiyuanhao1998/Retinexformer)

**Comment by ahmet-f-gumustas** [comment]
Hoping to Use : [DDRNet](https://github.com/Chenghao-Tan/DDRNet)

**Comment by yachty66** [comment]
Hoping to Use : [Flux Fill Dev](https://huggingface.co/black-forest-labs/FLUX.1-Fill-dev/tree/main)

**Comment by ishita-jainn** [comment]
Hoping to use : DeepLabV3+ and Segformer (mit-b4 backbone)
